danjones5071/shrapnel
# SHRPX-9 Create issues controller

:gift: closes #13.

### What was changed and why it was necessary
A controller was introduced for retrieving, creating, modifying, and deleted issues maintained by the application.

### How to test the changes
Perform the following steps to verify the changes:
1. Run the rspec tests with `rspec -fd` and verify that all tests pass.
2. Verify that the output sufficiently documents the controller's expected behavior.
3. Run the rails service and hit the following endpoints in Postman:
    - POST
    - GET
    - GET:id
    - PUT
    - DELETE


### Screenshots
**POST**
![POST](https://user-images.githubusercontent.com/42857836/65362729-91604300-dbd6-11e9-9aa7-1397492e9412.PNG)

**GET**
![GET](https://user-images.githubusercontent.com/42857836/65362737-96bd8d80-dbd6-11e9-91ad-56445ae8566d.PNG)

**GET:id**
![GET-id](https://user-images.githubusercontent.com/42857836/65362769-bf458780-dbd6-11e9-8c3b-e12677e4fa85.PNG)

**PUT**
![PUT](https://user-images.githubusercontent.com/42857836/65362779-ca001c80-dbd6-11e9-94f5-268111e87943.PNG)

**DELETE**
![DELETE](https://user-images.githubusercontent.com/42857836/65362787-cd93a380-dbd6-11e9-83a2-747d930f5a40.PNG)
